### ISSUE TITLE
avoid negations in questions

### ISSUE THREAD

**Issue by milahu** [8values/8values.github.io#97]
title says all

questions like "no authority should be left unquestioned" 
should be rephrased to "all authority must be questioned / verified"

this implies that "all authority can be fake" = mostly promise, seldom deliver
or more general "all labels can be fake"

damn, im so authoritarian : D
aka psychotic = fire or water = heartshape or ectomorph

of course political views corelate with personality = bodytype x gender x age ....
free market = economic freedom = classic class = mesomorph body = fire + earth

**Comment by Genora51** [comment]
Perhaps "should" works better than "must" as a change to make here:

i.e. rephrase it to `All authority should be questioned.`

**Comment by Genora51** [comment]
Just realised this is a duplicate of #64.
